internal/provider/organization_resource.go

```diff
@@ -83,6 +105,64 @@ func (r *OrganizationResource) Schema(ctx context.Context, req resource.SchemaRe
 				Default:  stringdefault.StaticString(""),
 			},
 		},
+
+		Blocks: map[string]schema.Block{
+			"group_sync": schema.SingleNestedBlock{
+				Attributes: map[string]schema.Attribute{
+					"field": schema.StringAttribute{
+						Optional: true,
+						MarkdownDescription: "The claim field that specifies what groups " +
+							"a user should be in.",
+						Validators: []validator.String{
+							stringvalidator.LengthAtLeast(1),
+						},
+					},
+					"regex_filter": schema.StringAttribute{
+						Optional: true,
+						MarkdownDescription: "A regular expression that will be used to " +
+							"filter the groups returned by the OIDC provider. Any group " +
+							"not matched will be ignored.",
+						Validators: []validator.String{
+							stringvalidator.LengthAtLeast(1),
+						},
+					},
+					"auto_create_missing": schema.BoolAttribute{
+						Optional: true,
+						MarkdownDescription: "Controls whether groups will be created if " +
+							"they are missing.",
+					},
+					"mapping": schema.MapAttribute{
+						ElementType:         types.ListType{ElemType: UUIDType},
+						Optional:            true,
+						MarkdownDescription: "A map from OIDC group name to Coder group ID.",
+						Validators: []validator.Map{
+							mapvalidator.ValueListsAre(listvalidator.ValueStringsAre(stringvalidator.Any())),
```

Comment:
The attribute is already `map(list(string))`, (UUIDType is also a string) so I don't think these validators are doing anything, though I assume they were added because `UUIDType` was broken.

Reply:
yeah, I was definitely just experimenting to see if I could jolt it into coercing the value out of that `unknown`